.github/scripts/check-orphaned-issues.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Script to find open issues in the camunda repository that are not assigned to any project
+# This helps ensure all issues are triaged and prioritized by the relevant teams
+
+ORG_NAME="camunda"
+REPO_NAME="camunda"
+PAGE_SIZE=100
+MAX_PAGES=50  # Safety limit to avoid infinite loops
+
+echo "Checking for open issues not assigned to any project in ${ORG_NAME}/${REPO_NAME}..."
+
+# Initialize variables for pagination
+ORPHANED_ISSUES=""
+ORPHANED_COUNT=0
+CURSOR="null"
+
+for ((page=1; page<=MAX_PAGES; page++)); do
+  # Build query with pagination cursor
+  if [[ "$CURSOR" == "null" ]]; then
+    CURSOR_ARG=""
+  else
+    CURSOR_ARG=", after: \"$CURSOR\""
+  fi
+
+  # Query for open issues in the repository
+  RESPONSE=$(gh api graphql -f query="
+    query {
+      repository(owner: \"$ORG_NAME\", name: \"$REPO_NAME\") {
+        issues(states: OPEN, first: $PAGE_SIZE$CURSOR_ARG) {
+          pageInfo {
+            hasNextPage
+            endCursor
+          }
+          nodes {
+            number
+            title
+            url
+            projectItems(first: 1) {
+              totalCount
+            }
+          }
+        }
+      }
+    }")
+
+  # Extract issues that have no project assignments (totalCount = 0)
+  PAGE_ISSUES=$(echo "$RESPONSE" | jq -r '
+    .data.repository.issues.nodes[]
+    | select(.projectItems.totalCount == 0)
```

Comment:
We also want to extract issues that are open and are _only_ assigned to projects that are not open